packages/github/src/interfaces.ts

```diff
@@ -1,39 +1,29 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
+// TODO: https://github.com/actions/toolkit/issues/291, ESLint chokes on the | syntax
+/* eslint-disable no-undef */
+import Webhooks from '@octokit/webhooks'
+type WebhookPayload =
+  | Webhooks.WebhookPayloadPush
```

Comment:
is the leading `|` intentional? seems like it would only be between each interface, but before the first interface.

does a leading `|` have a behavioral implication? or is it just an optionally allowed style?

Reply:
This is [valid typescript] (https://github.com/microsoft/TypeScript/issues/12071) and Prettier formats it this way. I actually coded it the way you suggested and prettier failed the check.

```
type a = b | c
```
is equivalent to 
```
type a = 
| b
| c
```

https://github.com/prettier/prettier/issues/6299